src/inspector/worker_agent.cc

```diff
@@ -115,6 +115,11 @@ DispatchResponse WorkerAgent::disable() {
   return DispatchResponse::OK();
 }
 
+DispatchResponse WorkerAgent::detach(const String& sessionId) {
+  workers_->Detached(sessionId);
```

Comment:
Do we really need a `detachedFromWorker` notification in response to a frontend request?

Reply:
Yes! It is how most domains (e.g. Target domain) in Chrome works. Notification means that worker session was detached for some reasons, one of the reasons might be explicit protocol request. It is easy to ignore this notification on frontend if needed.